app/assets/src/components/ui/controls/checkbox.scss

```diff
@@ -1,37 +1,45 @@
-@import "../../themes/colors";
+@import "~styles/themes/colors";
 
-.idseq-ui.input.checkbox {
+.checkbox {
   display: flex;
   position: relative;
-  padding: 0px 0px 0px 30px;
+  padding: 0px 0px 0px 0px;
   cursor: pointer;
   -webkit-user-select: none;
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
   height: 28px;
   align-items: center;
+  justify-content: center;
 
   input[type="checkbox"] {
-    position: absolute;
+    position: relative;
     opacity: 0;
     cursor: pointer;
+    clip: rect(0 0 0 0);
```

Comment:
Curious: why do we need this clip and the adjustments below?

Reply:
Hmm, turns out we don't. Thanks for catching this. (I had added these lines while working on this PR because the default, browser-styled checkbox had reappeared at some point.)